.github/scripts/cleanup_pr_body.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# ensure 1 argument is passed
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <pr_number>"
+    exit 1
+fi
+
+PR_NUMBER=$1
+OLD=/tmp/orig_pr_body.txt
+NEW=/tmp/new_pr_body.txt
+
+gh pr view --json body --template "{{.body}}" "${PR_NUMBER}" > "${OLD}"
+cp "${OLD}" "${NEW}"
+
+# Remove all lines after and including "**BEFORE SUBMITTING, PLEASE READ THE CHECKLIST BELOW AND FILL IN THE DESCRIPTION ABOVE**"
+sed -i '/\*\*BEFORE SUBMITTING, PLEASE READ THE CHECKLIST BELOW AND FILL IN THE DESCRIPTION ABOVE\*\*/,$d' "${NEW}"
+
+# Remove "FIX #xxxx (*link existing issues this PR will resolve*)"
+sed -i '/FIX #xxxx.*$/d' "${NEW}"
```

Comment:
There are a bunch of other ways an issue may be tagged as resolved by a PR. Not sure whether we can do this reliably.

Reply:
Why do we want to get rid of these messages? I agree it's going to be annoying to find a reliable way to remove this.

Reply:
Having literally "FIX #xxxx" other than the text from the PR template seems really bizarre?

The advantage to removing it is preventing garbage template text from making it into the commit log if merges start including the PR description / body.

Reply:
I thought about that. The challenge is that sometimes people only change the "FILL IN THE PR DESCRIPTION HERE" line, but leave the rest of the template in place.  That's why I split it into a few chunks.

Reply:
... wondering if you thought this was doing a pattern match for real issue/pr references instead of the literal text "#xxxx" ?

Reply:
I think removing the PR Checklist is already sufficient. Taking a look at the recent PRs, hardly any of them actually follow the "FIX #xxxx" from the template.

Reply:
That being said, I'm not opposed to a simple check to remove the common cases.

Reply:
> ... wondering if you thought this was doing a pattern match for real issue/pr references instead of the literal text "#xxxx" ?

In its current form, accidental removals should be quite unlikely since FIX has to be in all caps. But if we try to expand this, we have to be more careful not to accidentally remove references in the middle of a sentence. For example, removing the restriction on capitalization would cause this sentence to be partially removed:

```
This PR will fix #xxxx via method A.
             ^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Actually wait, I just realized that `xxxx` is a literal string. So it will only remove instances where the user doesn't edit that line at all. This is fine then.

Reply:
Yeah, for some reason I interpreted `FIX #xxxx` as `FIX #....`